recipes/treelite/bld.bat

```diff
@@ -0,0 +1,12 @@
+mkdir build
+cd build
+cmake .. -DENABLE_PROTOBUF=ON -GNinja
+if errorlevel 1 exit 1
+ninja install
+if errorlevel 1 exit 1
+cd ../python
+python setup.py install
```

Comment:
```suggestion
python -m pip install -e .
```

Reply:
Is `pip install -e` generally preferred? I thought it was just for development (since the package is made editable).

Reply:
Sorry you're right. Have edited the code suggestions to remove the `-e`. Otherwise using `pip` like so is recommended.

Reply:
Unfortunately, `pip install .` is currently broken in Treelite. It will take a while for me to try debugging it. Can we use `python setup.py install` for the time being? This command is well tested in the current CI, whereas `pip install .` is not.

Reply:
The Python Treelite package depends on the artifact `libtreelite.a` in the `build/` directory. According to the Pip documentation, `pip install .` will copy the content of `python/treelite` to a temporary directory first, so it will fail to locate the necessary `libtreelite.a`.

Reply:
Yeah using `setup.py` is fine. Though we need to add a couple of flags to it. These are to make sure `pip`/`setuptools` know that the Python library is being managed by a different package manager.

```suggestion
python setup.py install --single-version-externally-managed --record=record.txt
```

Reply:
I added `--single-version-externally-managed`.